frontend/admin/src/js/components/pool/PoolTable.tsx

```diff
@@ -6,13 +6,33 @@ import { notEmpty } from "@common/helpers/util";
 import { getLocale } from "@common/helpers/localize";
 import { FromArray } from "@common/types/utilityTypes";
 import Pending from "@common/components/Pending";
-import { getAdvertisementStatus } from "@common/constants/localizedConstants";
+import {
+  getAdvertisementStatus,
+  getPoolStream,
+} from "@common/constants/localizedConstants";
 import { commonMessages } from "@common/messages";
-import { GetPoolsQuery, Maybe, useGetPoolsQuery } from "../../api/generated";
+import {
+  GetPoolsQuery,
+  Maybe,
+  PoolStream,
+  useGetPoolsQuery,
+} from "../../api/generated";
 import Table, { ColumnsOf, tableEditButtonAccessor } from "../Table";
 import { useAdminRoutes } from "../../adminRoutes";
 
 type Data = NonNullable<FromArray<GetPoolsQuery["pools"]>>;
+type AccessorClassifications =
+  | (
+      | {
+          __typename?: "Classification" | undefined;
+          group: string;
+          level: number;
+        }
+      | null
+      | undefined
+    )[]
+  | null
+  | undefined;
```

Comment:
This can be simplified to `type AccessorClassifications = Data["classifications"];`, I believe